runtime/moonbase/src/lib.rs

```diff
@@ -799,7 +799,7 @@ construct_runtime! {
 		CrowdloanRewards: pallet_crowdloan_rewards::{Pallet, Call, Config<T>, Storage, Event<T>},
 		AuthorMapping: pallet_author_mapping::{Pallet, Call, Config<T>, Storage, Event<T>},
 		Proxy: pallet_proxy::{Pallet, Call, Storage, Event<T>},
-		Migrations: pallet_migrations::{Pallet, Storage, Config<T>, Event<T>},
+		Migrations: pallet_migrations::{Pallet, Storage, Config, Event<T>},
```

Comment:
`<T>` will probably be needed here to do anything interesting in a migration. It's no problem removing it for now though.

Reply:
why would it be needed?

Reply:
Nevermind, I was thinking of the `CommonMigrations`, you're right :+1: